modules/auxiliary/scanner/ipmi/ipmi_dumphashes.rb

```diff
@@ -91,6 +90,7 @@ def run_host(ip)
     add_socket(self.udp_sock)
 
     reported_vuln = false
+    session_succeeded = false
```

Comment:
Just confirming; Does this need to be reset for each user?

Reply:
Intent is to have it reset on per host basis. If at least one session succeeds for a host, then we keep on trying until all user names are tested.